aot-plugin/src/main/java/io/micronaut/gradle/aot/AbstractMicronautAotCliTask.java

```diff
@@ -42,6 +43,7 @@
 
 abstract class AbstractMicronautAotCliTask extends DefaultTask implements OptimizerIO {
 
+    private final Property<JavaLauncher> javaLauncher;
```

Comment:
:x: Use `public abstract Property<JavaLauncher> getJavaLauncher()` instead. The implementation is automatically created by Gradle.

Also, annotated it with `@Input`.